pkgs/build-support/build-typst-package.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenvNoCC }:
+{
+  nativeBuildInputs ? [ ],
+  buildInputs ? [ ],
+  meta ? { },
+  ...
+}@attrs:
+let
+  typstPkgPrefix = "typst-package";
+in
+stdenvNoCC.mkDerivation (
+  attrs
+  // (with attrs; rec {
```

```suggestion
  // (with attrs; {
```
This doesn't need to be recursive.